content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Package and deploy your models with MLEM
+date: 2022-05-24
+description: >
+  Introducing MLEM -- Machine Learning Engineering Management for Model
+  Deployment and Management.
+descriptionLong: >
+  We’re excited to announce the launch of our latest open source offering,
+  [MLEM](https://mlem.ai)! MLEM is a tool that automatically extracts meta
+  information like environment and frameworks from models and standardizes that
+  information into a human-readable format within Git. ML teams can then use the
+  model information for deployment into downstream production apps and services.
+  MLEM easily connects to solutions like Heroku to dramatically decrease model
+  deployment time.
+picture: 2022-05-24/mlem-rocket.png
+author: aguschin
+# commentsUrl: TODO
+tags:
+  - Machine Learning
+  - Deployment
+  - Model Registry
+  - MLOps
+---
+
+We built MLEM to address issues that MLOps teams have around managing model
+information as they move them from training and development to production and,
+ultimately, retirement. MLEM is meant to help teams automate the collection of
+information around how the model was trained, what the model is for, and
+operational requirements around deployment.
+
+Just like all our [other](https://dvc.org) [tools](https://cml.dev), MLEM uses
```

Comment:
Instead of doing these two links, maybe we should send to... Ok nevermind.  I thought we have a product page at iterative.ai, but it's just a drop - down.  cc: @jurv11 @julieg18, we should add this to the website list if we don't have it on there yet.  There's the pricing page which shows all the tools, but that's not where we would want to send people in this case.